### PR TITLE
Signed in home

### PR DESCRIPTION
**Pull request by dswcodes0**

### CODE REVIEW

**Comment by dswcodes0** [comment]
This is the SignedInHome branch and is finished with the html. it is mostly placeholders for the inserted javascript later

**Review by charliegriffinfacebook — CHANGES_REQUESTED** [review]
Good diff size!

Please include a test plan so I know what this code does and how it works.

**Comment by dswcodes0** [comment]
i finished the signedinhome page, this is mainly placeholders so it is very bare. here it is:
Screenshot 2023-07-07 at 12 09 08 PM

**Comment by annagaofacebook** [comment]
Project question -- what does "Search" do vs "Ask me a question"?

File: movie-app/src/SignedInHome/SignedInHome.jsx [hunk 1]
```diff
@@ -4,8 +4,31 @@ import './SignedInHome.css';
...
+      <div className="chatbot">
+        <div className="chatbot-img">
+          <h1>How can I help?</h1>
+          <img src="../chatbot.png" alt="" />
+          <input type="text" placeholder="Ask me a question..." />
+          <input type="submit" value="Submit" />
+        </div>
       </div>
+      <div id="movies-grid" className="container">
+      {/* Placeholder for movie elements */}
```
Comment (annagaofacebook) [inline movie-app/src/SignedInHome/SignedInHome.jsx:16]: Nice, these are good comments that specify what should go here in the future (once you get to them). Good way to keep track of TO-DOs.

**Comment by dswcodes0** [comment]
> Project question -- what does "Search" do vs "Ask me a question"?

search will be in the navbar. when it is styled it will be much smaller. ask me a question will be in the main content area and be much bigger. also, the images will be smaller and bigger for the navbar and the main content, respectively. @annagaofacebook

**Pull request merged**